### ISSUE TITLE
Issue with kartik-editablegrid

### ISSUE THREAD

**Issue by SplendornetTech** [kop/yii2-scroll-pager#102]
Hi,
First off all, great extension, makes life far easy for many yii developers. Kudos to you.
I am using Kartik-editable grid with it's own pjax event like below

GridView::widget([
            'dataProvider' => $dataProvider,
            'filterModel' => $searchModel,
            'pjax' => true,
            'pager' => [
                'class' => \kop\y2sp\ScrollPager::className(),
                'container' => '.grid-view tbody',
                'item' => 'tr',
                'paginationSelector' => '.grid-view .pagination',
                'triggerTemplate' => '<tr class="ias-trigger"><td colspan="100%" style="text-align: left"><a style="cursor: pointer">{text}</a></td></tr>',
                'eventOnRendered' => 'function() {showColumns();}',
            ],
                        [
                    'class' => 'kartik\grid\EditableColumn',
                    'header' => 'Delivery Date',
                    'attribute' => 'delivery_date',
                    'format' => 'html',
                    'value' => function ($data) {
                        if (!empty($data->tempOaDetails) && $data->is_temp) {
                            return ($data->tempOaDetails->delivery_date=='0000-00-00')? '' : date("d-m-Y", strtotime($data->tempOaDetails->delivery_date));
                        } else {
                            $OaDetails = app\models\OaDetails::find()->where(['oa_key' => $data->oa_key])->one();
                            return ($OaDetails->delivery_date=='0000-00-00')? '' : date("d-m-Y", strtotime($OaDetails->delivery_date));
                        }
                    },
                    'headerOptions' => ['class' => 'setWidth delivery_date', 'style' => 'color:#3c8dbc;text-align:left;'],
                    'contentOptions' => ['class' => 'delivery_date', 'style' => 'text-align:left'],
                    'editableOptions' => function ($data) use ($accessLevel,$action, $sales_target_year){
                        if(($accessLevel::checkAccess('Add Order Acceptance') && 
                            $action != 'oa-revision-index' && 
                            \app\models\TempOaType::getSalesTargetYear20() == $sales_target_year || 
                            User::isAdminUser())){
                            $date = '';
                            if (!empty($data->tempOaDetails) && $data->is_temp) {
                                $date =  ($data->tempOaDetails->delivery_date=='0000-00-00')? '' : date("d-m-Y", strtotime($data->tempOaDetails->delivery_date));
                            } else {
                                $OaDetails = app\models\OaDetails::find()->where(['oa_key' => $data->oa_key])->one();
                                $date = ($OaDetails->delivery_date=='0000-00-00')? '' : date("d-m-Y", strtotime($OaDetails->delivery_date));
                            }
                            $date = Helper::convertDateTimeInView($date);
                            $date= ($date == '0000-00-00' || $date == '') ? date('d-m-yy') : $date;
                            return [
                                'name'=>'Delivery Date', 
                                'asPopover' => true,
                                'header' => 'Specify/Change Delivery Date',
                                'attribute' => 'delivery_date',
                                'size'=>'ms',
                                'inputType' => Editable::INPUT_DATE,
                                'value' => $date,
                                'options'=>[
                                    'options'=>['placeholder'=>'Select Date','value'=>$date,'data-key'=>$data->oa_key],
                                    'pluginOptions' => [
                                        'autoclose'=>true,
                                        'format' => 'dd-mm-yyyy'
                                    ]
                                ],
                                'editableValueOptions' => [ 'disabled' => $data->oa_main_status != 'Completed' ? false : true],
                            ];
                        } else return;
                    }
                ], 

When I hit the load more option, everything works fine except I am not able to edit the date in the editable column for the newely loaded rows. I do understand that the events for these rows is lost, my question is how to rebind them again? I know this might be beyond your plugin and probably need to ask Kartik about it, but just in case you have overcome this issue, please do suggest a way.

Regards
Aditya

**Comment by akula222** [comment]
тоже проблема с катриком

**Comment by LandDevTeam** [comment]
Hey
@SplendornetTech did you get any solution??
I am stack same kind of issue.
Thanks in advance.

**Comment by akula222** [comment]
+++++++++++++++++++++++++++

**Comment by LandDevTeam** [comment]
> +++++++++++++

Do you want to say something??

**Comment by akula222** [comment]
yii2-scroll-pager and kartik\grid\EditableColumn problem :(

**Comment by ExpandXSoln** [comment]
Yes there is a solution available to this, check my comments on the kartik editable grid git issue list for the similar issue reported there too
